### PR TITLE
Remove WORKAROUND_ISSUE_2298

### LINKED ISSUE #1050: Remove WORKAROUND_ISSUE_2298.
This should  enable some tests, e.g. https://github.com/ROCmSoftwarePlatform/MIOpen/blob/dd153b6eb77f20dc68c90bdf3cfa1733b67b0e12/test/CMakeLists.txt#L1364-L1370

See https://github.com/AMDComputeLibraries/MLOpen/issues/2298#issuecomment-858136003 for more info.

### PR DESCRIPTION
**Pull request by averinevg**
Removes WORKAROUND_ISSUE_2298.

Fixes #1050

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -173,10 +173,10 @@ else()
...
 endif()
 
-if(MIOPEN_TEST_GFX908)
-    if(WORKAROUND_ISSUE_1064)
-        list(APPEND SKIP_TESTS test_dropout)
-    endif()
+if(MIOPEN_TEST_GFX908)
+    if(WORKAROUND_ISSUE_1064)
+        list(APPEND SKIP_TESTS test_dropout)
+    endif()
```
Comment (atamazov) [inline test/CMakeLists.txt:179]: This is whitespace-only changes or something else useless. Please revert this.

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -1382,7 +1382,6 @@ endif()
 # Add here regression tests that should be run only on Vega10/20 and only with FP16.
 add_custom_test(test_regression_half_vega FLOAT_DISABLED HALF_ENABLED GFX908_DISABLED
```
Comment (atamazov) [inline test/CMakeLists.txt:1383]: The workaround comment has been deleted, but what about the test itself? Let's actually enable the test for gfx908:
```suggestion
add_custom_test(test_regression_half_vega FLOAT_DISABLED HALF_ENABLED GFX908_ENABLED
```

**Review by atamazov — DISMISSED** [review]

**Comment by atamazov** [comment]
@averinevg Please set the same labels as #1050 has, thanks.

File: test/CMakeLists.txt (hunk 1 above)
Comment (averinevg) [reply test/CMakeLists.txt]: Done, thanks.

File: test/CMakeLists.txt [hunk 3]
```diff
@@ -1380,9 +1380,8 @@ if(MIOPEN_TEST_FLOAT)
 endif()
 
 # Add here regression tests that should be run only on Vega10/20 and only with FP16.
-add_custom_test(test_regression_half_vega FLOAT_DISABLED HALF_ENABLED GFX908_DISABLED
+add_custom_test(test_regression_half_vega FLOAT_DISABLED HALF_ENABLED GFX908_ENABLED
```
Comment (shurale-nkn) [inline test/CMakeLists.txt:1383]: Now the name and comment are misleading

**Review by shurale-nkn — DISMISSED** [review]
LGTM

**Comment by junliume** [comment]
gfx908 CI temporarily blocked by a public issue: https://github.com/RadeonOpenCompute/ROCm/issues/1538 
Will restart and update when the GPG issue is resolved
@okakarpa could you update the nodes with new GPG key

**Comment by junliume** [comment]
> gfx908 CI temporarily blocked by a public issue: [RadeonOpenCompute/ROCm#1538](https://github.com/RadeonOpenCompute/ROCm/issues/1538)
> Will restart and update when the GPG issue is resolved
> @okakarpa could you update the nodes with new GPG key

Should have been resolved with #1079 merged and triggered another CI round

**Review by shurale-nkn — APPROVED** [review]
LGTM

**Pull request merged**